src/backend/db/models.py

```diff
@@ -128,7 +128,7 @@ class Mirror(Base):
     ip = Column(String, nullable=False)
     latitude = Column(Float, nullable=False)
     longitude = Column(Float, nullable=False)
-    is_expired = Column(Boolean, nullable=False, default=False)
+    status = Column(String, nullable=False)
```

Comment:
Did you make db migrations?

Reply:
sqlite cannot change column type so no I didn't.  Do we want one that simply removes the is_expired col and adds status?

Reply:
I did migration to postgres which can do it)

Reply:
Never mind. I just mix up services.